android/app/build.gradle

```diff
@@ -163,9 +163,9 @@ dependencies {
     androidTestImplementation 'com.android.support.test.espresso:espresso-contrib:3.0.2'
     androidTestImplementation 'androidx.test:rules:1.3.0-alpha03'
 
-    implementation 'network.mysterium:mobile-node:0.32.0'
+    // implementation 'network.mysterium:mobile-node:0.31.3'
     // Comment network.mysterium:mobile-node and replace with your local path to use local node build.
-    // implementation files('/Users/anjmao/go/src/github.com/mysteriumnetwork/node/build/package/Mysterium.aar')
+    implementation files('/Users/anjmao/go/src/github.com/mysteriumnetwork/node/build/package/Mysterium.aar')
```

Comment:
@anjmao this ref probably shouldn't be in the master?

Reply:
Yes, I'm waiting for node repo PR first and when will update this PR.